### PR TITLE
Add information for external snapshots (bsc#1125660)

### PR DESCRIPTION
**Pull request by liayan**
### Description
Add information for external snapshots

*Are backports required?*

- [Y] To maintenance/SLE12SP3
- [Y] To maintenance/SLE12SP4
- [Y] To maintenance/SLE15SP0

### CODE REVIEW

File: xml/libvirt_managing.xml [hunk 1]
```diff
@@ -631,8 +631,10 @@
        made read-only, while a new qcow2 file is created to hold the changes.
        The original file is sometimes called a 'backing' or 'base' file, while
        the new file with all the changes is called an 'overlay' or 'derived'
-       file. External snapshots are useful when performing backups of
-       &vmguest;s. For more information on external snapshots in &qemu;, refer
+       file. External snapshots are useful when performing backups of &vmguest;s.
+       However, External snapshots could not be supported by &vmm;, and
```
Comment (taroth21) [inline xml/libvirt_managing.xml]: I would rephrase this sentence to: 'However, external snapshots are not supported by &vmm;, and cannot be deleted by <command>virsh</command> directly.' (there also was a small typo in 'directly')

File: xml/libvirt_managing.xml [hunk 2]
```diff
@@ -885,8 +887,8 @@ Metadata:       yes
    <sect3 xml:id="sec.libvirt.managing.snapshots.virsh.delete">
     <title>Deleting a Snapshot</title>
     <para>
-     To delete a snapshot of a &vmguest; and restore the disk space it
-     occupies, use the <literal>snapshot-delete</literal> command:
+     External snapshots could be deleted here. To delete an internal snapshot of a &vmguest;
```
Comment (taroth21) [inline xml/libvirt_managing.xml]: After looking at bsc#1125660 and the article that was referenced there, I think this sentence should be:
'External snapshots cannot be deleted with ```<command>virsh</command>```.'

Comment (liayan) [reply xml/libvirt_managing.xml]: Really appreciate the review, I changed the parts you mentioned, please have a look and let me know if have further questions.

File: xml/libvirt_managing.xml (hunk 1 above)
Comment (liayan) [reply xml/libvirt_managing.xml]: Changed as request.

**Comment by taroth21** [comment]
@liayan : Thanks for the modifications! As you indicated above that the changes should go to the documentation for SLE15SP0, SLE12SP4 and SLE12SP3, too, I will take care of  backporting them the those maintenance branches.
